src/inc/quic_platform_winuser.h

```diff
@@ -880,6 +966,9 @@ CxPlatCqeGetSqe(
     _In_ const CXPLAT_CQE* cqe
     )
 {
+    if (cqe->lpOverlapped == NULL) {
+        return (CXPLAT_SQE*)cqe->lpCompletionKey;
```

Comment:
See the comment above - I think that if you set the sqe as the APC context, it will be mapped to `lpOverlapped` here.